apps/web/src/server/views/applications/case/project-information.njk

```diff
@@ -1,11 +1,170 @@
+						],
+						[
+							{ text: "Project location" },
+							{ text: application.geographicalInformation.locationDescription },
+							{ html: writeChangeLink({ editStepUrl: 'project-location' }), classes: 'text-align--right pins-table__cell--s' }
+						],
+						[
+							{ text: "Grid references" },
+							{ html: gridReferences },
+							{ html: writeChangeLink({ editStepUrl: 'grid-references' }), classes: 'text-align--right pins-table__cell--s' }
+						],
+						[
+							{ text: "Regions(s)" },
+							{ text: regionNames },
+							{ html: writeChangeLink({ editStepUrl: 'regions' }), classes: 'text-align--right pins-table__cell--s' }
+						],
+						[
+							{ text: "Map zoom level" },
+							{ text: application.geographicalInformation.mapZoomLevel.displayNameEn },
+							{ html: writeChangeLink({ editStepUrl: 'zoom-level' }), classes: 'text-align--right pins-table__cell--s' }
+						]
+					]
+				})
+			}
+		},
+		{
+			heading: { text: 'Applicant information' },
+			content: {
+				html: govukTable({
+					firstCellIsHeader: true,
+					rows: [
+						[
+							{ text: 'Organisation name' },
+							{ text: applicantFirstOrganisationName },
+							{ html: writeChangeLink({ editStepUrl: 'applicant-organisation-name' }), classes: 'text-align--right pins-table__cell--s' }
+						],
+						[
+							{ text: "Contact name" },
+							{ text: writeFullName([application.applicants[0].firstName, application.applicants[0].middleName, application.applicants[0].lastName] )},
+							{ html: writeChangeLink({ editStepUrl: 'applicant-full-name' }), classes: 'text-align--right pins-table__cell--s' }
+						],
+						[
+							{ text: "Address" },
+							{ text: writeFullAddress(application.applicants[0].address) },
+							{ html: writeChangeLink({ editStepUrl: 'applicant-address' }), classes: 'text-align--right pins-table__cell--s' }
+						],
+						[
+							{ text: "Website" },
+							{ text: applicantFirstWebsite },
+							{ html: writeChangeLink({ editStepUrl: 'applicant-website' }), classes: 'text-align--right pins-table__cell--s' }
+						],
+						[
+							{ text: "Email address" },
+							{ html: applicantFirstEmail },
+							{ html: writeChangeLink({ editStepUrl: 'applicant-email' }), classes: 'text-align--right pins-table__cell--s' }
+						],
+						[
+							{ text: "Telephone number" },
+							{ text: applicantFirstPhoneNumber },
+							{ html: writeChangeLink({ editStepUrl: 'applicant-telephone-number' }), classes: 'text-align--right pins-table__cell--s' }
+						]
+					]
+				})
+			}
+		}
+		]
+	}) }}
 
-	<hr class="govuk-section-break govuk-section-break--l govuk-section-break--visible">
 {% endblock %}
+
+{% macro writeChangeLink(params) %}
+	{% set userIsInspector = domainType === 'inspector' %}
+
+	{% if not userIsInspector %}
+		{% set baseUrl = '/applications-service/case/' + applicationId +'/edit/' %}
```

Comment:
please use the url filter.
should be `'applications-edit|url({applicationId, step})`

Reply:
@lutamKC yes! I was just thinking that, and then I saw this comment!  Will do!